data/usmangt.json

```
@@ -0,0 +1,31 @@
+{
+	"name": "Syed Usman Ahmad",
+	"type": "personal",
+	"bio": "Open Source contributor | DevRel | Working at **Grafana Labs**",
+
+	"tags": [
+		"Open Source",
+		"Linux",
+		"Kubernetes",
+		"Docker",
+		"Grafana",
+		"DevOps",
+		"API",
+		"Cloud (SaaS)",
+		"Database",
+		"Technical Support"
+	],
+
+
+	"links": [{
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/syed-usman-ahmad-b1415515/",
+			"icon": "FaLinkedIn"
```

You probably be using an invalid icon here. It should be `FaLinkedin`

```suggestion
			"icon": "FaLinkedin"
```